solver/llbsolver/bridge.go

```diff
@@ -177,6 +177,9 @@ func (rp *resultProxy) Definition() *pb.Definition {
 }
 
 func (rp *resultProxy) Provenance() interface{} {
+	if rp.provenance == nil {
+		return nil
+	}
```

Comment:
typed `nil` strikes again :cry:

Reply:
Think it _might_ be related to this change [`e46296c` (#3949)](https://github.com/moby/buildkit/pull/3949/commits/e46296c21616ad4c84322b09d60094108ed5f40c#diff-2fc2e99eea0899fac8a14da1878d67aa8520bed09be4de598226d4d29450d6c8R282)